src/pythonTestAdapter.ts

```diff
@@ -136,9 +149,21 @@ export class PythonTestAdapter implements TestAdapter {
         try {
             this.testStatesEmitter.fire({ type: 'started', tests });
             const config = await this.configurationFactory.get(this.workspaceFolder);
+
+            let collector: LoggingOutputCollector | undefined;
```

Comment:
nit: if you extract initialization to a function you can use const and code might be a bit cleaner.

Reply:
I have extracted it to function